docs/deployment/upgrade.md

```diff
@@ -8,53 +8,60 @@ hide_title: true
+3. **Migrate lower environments first:** Migrate your "lowest" environment first and test it to ensure correctness before performing migration steps in each successively higher environment.  For example, in the EDCloud progress from a sandbox environment to UAT to SIT to PROD.
+4. **Migrate during appropriate times:** Choose a time to migrate when support is more likely to be available in case you encounter problems, such as when you are most likely to quickly find help in the #cumulus-internal Slack channel.  Prefer earlier in the week over later in the week (particularly avoiding Fridays, if possible).
 
-## Updating Cumulus version
+## Updating Cumulus Version
 
 To update your Cumulus version:
 
-1. Find the desired release on the [Cumulus releases page](https://github.com/nasa/cumulus/releases)
-2. Update the `source` in your Terraform deployment files **for each of [your Cumulus modules](./components.md#available-cumulus-components)** by replacing `vx.x.x` with the desired version of Cumulus:
+1. Find the desired release on the [Cumulus Releases] page
+2. Update the `source` in your Terraform deployment files **for each of [your Cumulus modules](./components.md#available-cumulus-components)** by replacing `vx.x.x` with the desired version of Cumulus.  For example, here's the
+entry from the `data-persistence` module:
 
     `source = "https://github.com/nasa/cumulus/releases/download/vx.x.x/terraform.zip//tf-modules/data-persistence"`
 
 3. Run `terraform init` to get the latest copies of your updated modules
 
-## Update data persistence resources
+## Update Data Persistence Resources
 
-**Reminder:** Remember [to initialize Terraform](./README.md#initialize-terraform) if necessary.
+**Reminder:** Remember to [initialize Terraform](./README.md#initialize-terraform) if necessary.
 
-From the directory of your `data-persistence` deployment module (e.g. `data-persistence-tf`):
+From the directory of your `data-persistence` deployment module (e.g., `data-persistence-tf`):
 
 ```bash
-  $ AWS_REGION=<region> \ # e.g. us-east-1
-      AWS_PROFILE=<profile> \
-      terraform apply
+$ AWS_REGION=<region> \ # e.g. us-east-1
+    AWS_PROFILE=<profile> \
+    terraform apply
 ```
 
-## Update Cumulus resources
+## Update Cumulus Resources
 
 **Reminder:** Remember [to initialize Terraform](./README.md#initialize-terraform) if necessary.
 
-From the directory of your `cumulus` deployment module (e.g. `cumulus-tf`):
+From the directory of your `cumulus` deployment module (e.g., `cumulus-tf`):
 
 ```bash
-  $ AWS_REGION=<region> \ # e.g. us-east-1
-      AWS_PROFILE=<profile> \
-      terraform apply
+$ AWS_REGION=<region> \ # e.g. us-east-1
+    AWS_PROFILE=<profile> \
+    terraform apply
 ```
+
```

Comment:
Maybe add sections for updating the dashboard if applicable to take advantage of latest API functionality and a section for verifying your Cumulus deployment.

Reply:
I don't know what that would look like, so I'm happy to take suggestions.

Reply:
Something along the lines of: After deployment and migrations, run tests to verify your workflows are working before promoting the changes to higher environments. A good test to run would be your discovery and ingest workflows to verify ingest has not been impacted by the deployment.

Reply:
Ah, right. I added such details. I thought this comment might be referring to something else. Please see my other changes, which address this.

Reply:
Yep can you just add it as a step here please? You can refer back to the part about smoke tests.